collector/nginx_plus.go

```diff
@@ -110,6 +112,30 @@ func NewNginxPlusCollector(nginxClient *plusclient.NginxClient, namespace string
 			"records_pending": newStreamZoneSyncZoneMetric(namespace, "records_pending", "The number of records that need to be sent to the cluster"),
 			"records_total":   newStreamZoneSyncZoneMetric(namespace, "records_total", "The total number of records stored in the shared memory zone"),
 		},
+		locationZoneMetrics: map[string]*prometheus.Desc{
+			"requests":      newLocationZoneMetric(namespace, "requests", "Total client requests", nil),
+			"responses_1xx": newLocationZoneMetric(namespace, "responses", "Total responses sent to clients", prometheus.Labels{"code": "1xx"}),
+			"responses_2xx": newLocationZoneMetric(namespace, "responses", "Total responses sent to clients", prometheus.Labels{"code": "2xx"}),
+			"responses_3xx": newLocationZoneMetric(namespace, "responses", "Total responses sent to clients", prometheus.Labels{"code": "3xx"}),
+			"responses_4xx": newLocationZoneMetric(namespace, "responses", "Total responses sent to clients", prometheus.Labels{"code": "4xx"}),
+			"responses_5xx": newLocationZoneMetric(namespace, "responses", "Total responses sent to clients", prometheus.Labels{"code": "5xx"}),
+			"discarded":     newLocationZoneMetric(namespace, "discarded", "Requests completed without sending a response", nil),
+			"received":      newLocationZoneMetric(namespace, "received", "Bytes received from clients", nil),
+			"sent":          newLocationZoneMetric(namespace, "sent", "Bytes sent to clients", nil),
+		},
+		resolverMetrics: map[string]*prometheus.Desc{
```

Comment:
You can find descriptions of the metrics here: https://nginx.org/en/docs/http/ngx_http_api_module.html#def_nginx_resolver_zone

So use those as a guide while following the style of the other metric descriptions.

Reply:
thanks